src/style.css

```diff
@@ -0,0 +1,22 @@
+body * {
+  margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+}
+
+h2,
+a {
+  font-size: 15px;
+  font-weight: normal;
+  margin-top: 1em;
+  margin-bottom: 1em;
+}
+
+a {
+  color: green;
+}
```

Comment:
Please use classes to add styles to your document.
However, default font styles can be set in `body` selector.

Also, dont use `*` selector for now.
Its correct usage will be explained further in the program.